__tests__/index-test.js

```diff
@@ -422,3 +422,25 @@ describe('when used with typescript', () => {
     expect(actual).toEqual(`Ember.addObserver();`);
   });
 });
+
+describe('when used with babel-plugin-istanbul', () => {
+  // babel-plugin-istanbul won't run on <= Node 6
+  const majorVersion = parseInt(process.version.match(/^v(\d+)\./)[1], 10);
+  const runOrSkip = majorVersion > 6 ? it : it.skip;
+
+  runOrSkip('throws an exception', () => {
+    let source = `
+      import EmberObject from '@ember/object';
+      import Evented from '@ember/object/evented';
+
+      export default class TestObject extends EmberObject.extend(Evented) {};
+    `;
+
+    expect(() => {
+      babel7.transformSync(source, {
+        filename: 'istanbul-should-cover.js',
+        plugins: [require('babel-plugin-istanbul'), Plugin],
+      });
+    }).toThrow(/Container is falsy/i);
```

Comment:
Can you tweak this to something like:

```suggestion
    expect(() => {
      babel7.transformSync(source, {
        filename: 'istanbul-should-cover.js',
        plugins: [require('babel-plugin-istanbul'), Plugin],
      });
    }).toMatchInlineSnapshot();
```

Basically, we **don't** want it to throw...

Reply:
To make sure I'm tracking, the idea is to provide feedback that's a bit more useful than "it did/not throw"?

Reply:
Right, exactly. Basically `expect().toThrow()` is not what we want here, we want to say it **does not** throw (IOW, this PR should be failing not passing). Using `.toMatchInlineSnapshot` just lets us confirm that it emits _something_ that we can visually verify "looks correct".